docs/coding-guidelines/javascript.md

```diff
@@ -420,6 +402,18 @@ const myStr = "You're amazing just the way you are.";
 const component = <div className="post"></div>;
 ```
 
+[ES2015 template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) are available as an alternative to string concatenation when including variables in a string.
```

Comment:
>There is this obscure usage of tag functions with template literals.

I personally feel we should avoid advocating their usage. That is... unless we ever adopt Relay 😉  ([reference](https://facebook.github.io/relay/docs/api-reference-relay-ql.html))